e2e/__tests__/__snapshots__/detectOpenHandles.ts.snap

```diff
@@ -9,8 +30,20 @@ This usually means that there are asynchronous operations that weren't stopped i
 `;
 
 exports[`prints out info about open handlers 1`] = `
-Jest has detected the following 1 open handle potentially keeping Jest from exiting:
+Jest has detected the following open 3 handles potentially preventing Jest from exiting:
+Of them 1 was collected within 100ms of the tests completing.
+These are sometimes useful to look at as they might have spawned other handles that remain open, but that we have lost the origin of.
+Uncollected handles: 
+  ●  DNSCHANNEL
+
 
+
+  ●  TCPSERVERWRAP
+
+
```

Comment:
Can we remove these extra newlines between handles?

Reply:
just pushed it 🙂

Reply:
you were too quick though, so I force pushed. sorry about duplicate diffs 😬